### PR TITLE
Adding information for version functionality

### PR DESCRIPTION
**Pull request by sylvaincombes**
I had a quite hard time to collect informations and make working the version functionality of the bundle.
I think it requires more detailed informations.
Here is my proposal, feel free to retouch my input to better english / informations :)

### CODE REVIEW

File: Resources/doc/format_listener.rst [hunk 1]
```diff
@@ -124,6 +124,35 @@ The matched version is set as a Request attribute with the name ``version``,
...
 
 .. _`this JMS Serializer article`: http://jmsyst.com/libs/serializer/master/cookbook/exclusion_strategies#versioning-objects
+
+That's it, it should work now.
+
+If you have to verify if the version is correctly catched you can use something like :
+
+.. code-block:: php
+
+        if($this->container->get('fos_rest.version_listener')) {
```
Comment (lsmith77) [inline Resources/doc/format_listener.rst]: missing space .. ie. ``if (``

File: Resources/doc/format_listener.rst [hunk 2]
```diff
@@ -124,6 +124,35 @@ The matched version is set as a Request attribute with the name ``version``,
...
 automatically in the ``ViewHandler``.
 
-See `this JMS Serializer article`_ for details.
+To make the version mechanism working :
+
+1 - The client must pass the requested version in his header like this :
+
+.. code-block:: yaml
+
+    Accept:application/json;version=1.0
```
Comment (lsmith77) [inline Resources/doc/format_listener.rst]: you should reference http://symfony.com/doc/master/bundles/FOSRestBundle/format_listener.html#media-type-version-extraction as the format is actually configurable

**Comment by lsmith77** [comment]
great thanks! two minor comments .. once these are fixed, I will merge and release 1.6.0 stable

**Comment by sylvaincombes** [comment]
:+1: thank you

**Pull request merged**